File: src/weaponinventorymod/gui/StockReviewTradeGroup.java
package weaponinventorymod.gui;

enum StockReviewTradeGroup {
    BUYING("Buying"),
    SELLING("Selling");

    private final String label;

    StockReviewTradeGroup(String label) {
        this.label = label;
    }

    String getLabel() {
        return label;
    }
}

File: src/main/kotlin/weaponsprocurement/ui/stockreview/trade/StockReviewTradeGroup.kt
package weaponsprocurement.ui.stockreview.trade

import weaponsprocurement.ui.stockreview.actions.*
import weaponsprocurement.ui.stockreview.state.*
import weaponsprocurement.ui.stockreview.rows.*
import weaponsprocurement.ui.stockreview.tooltips.*
import weaponsprocurement.ui.stockreview.rendering.*
import weaponsprocurement.ui.stockreview.trade.*

import weaponsprocurement.ui.*

enum class StockReviewTradeGroup(val label: String) {
    BUYING("Buying"),
    SELLING("Selling"),
}